tests/platform_tests/conftest.py

```diff
@@ -76,6 +76,12 @@ def xcvr_skip_list(duthosts):
             logging.debug(
                 "hwsku.json absent or port_type for interfaces not included for hwsku {}".format(hwsku))
 
+        # No hwsku.json for Arista-7050-QX-32S/Arista-7050QX-32S-S4Q31
```

Comment:
@byu343 Why there is a change in behavior?

Reply:
We decided to remove hwsku.json (in the change https://github.com/sonic-net/sonic-buildimage/pull/17253) because 1) the correct port index should be 1,2,3,4 for the SFP ports under this hwsku Arista-7050-QX-32S/Arista-7050QX-32S-S4Q31; 2) but same logical ports with a different breakout mode under another hwsku have to use port index 5,5,5,5, which is in https://github.com/sonic-net/sonic-buildimage/blob/master/device/arista/x86_64-arista_7050_qx32s/platform.json#L215; 3) removing hwsku.json will fall back to use port_config.ini, where we can set the port index as 1,2,3,4 while not affecting other hwskus.

The xcvr_skip_list here was to skip SFP based on the RJ45 setting in hwsku.json, but since the hwsku.json of Arista-7050-QX-32S/Arista-7050QX-32S-S4Q31 has to be removed, we decided to add the skip for the hwskus in the test code here.